docs/ref/contrib/admin/index.txt

```diff
@@ -2749,6 +2749,61 @@ The presence of the ``admin_password_reset`` named URL will cause a "forgotten
 your password?" link to appear on the default admin log-in page under the
 password box.
 
+.. _admin-logentries:
+
+``LogEntry`` objects
+====================
+
+.. class:: django.contrib.admin.models.LogEntry
+
+    The ``LogEntry`` class represents your activity in the admin interface:
```

Comment:
I'd say "tracks additions, changes, and deletions of objects done through the admin interface."